crates/biome_js_analyze/src/assist/source/use_sorted_keys.rs

```diff
@@ -44,14 +44,14 @@ declare_source_rule! {
     /// ## Examples
     ///
     /// ```js,expect_diff
-    /// {
+    /// const obj = {
```

Comment:
This fixes some diagnostics that were previously not caught by `just lint-rules`. Now they get successfully detected, because the CodeBlock parser now supports `expect_diff` the same way as the website.